### PR TITLE
layout_html-form

### PR DESCRIPTION
**Pull request by AnastasiiaSemenovaa**
- [DEMO LINK](https://AnastasiiaSemenovaa.github.io/layout_html-form/)
- [TEST REPORT LINK](https://AnastasiiaSemenovaa.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,17 +1,135 @@
...
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input 
+          type="text" 
+          name="surname"
+           autocomplete="off" 
+          >
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
          <input 
            type="text" 
            name="surname"
            autocomplete="off" 
          >
```

Comment (Volodymyr-Melnyk-Fron10der) [reply src/index.html]: check it throughout your code

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,135 @@
...
+        </label>
+        <label class="label">
+          Name:
+          <input 
+          type="text" 
+          name="name" 
+          autocomplete="off" 
+          >
+        </label>
+        <label class="label">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]: add one empty line between large siblings to make code more readable

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,135 @@
...
+            > No
+          </label>
+        </label>
+        <label class="label">
+          What is your favorite color? <input type="color" name="color" >
+        </label>
+        <label class="label">
+          What time do you go to bed? <input type="time" name="bedTime" >
+        </label>
+        <label class="label">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
        </label>
        
        <label class="label">
          What is your favorite color? <input type="color" name="color" >
        </label>
        
        <label class="label">
          What time do you go to bed? <input type="time" name="bedTime" >
        </label>
        
        <label class="label">
```

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Cool, just a few syntax issues I see

File: src/index.html [hunk 4]
```diff
@@ -1,17 +1,152 @@
...
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth: 
+          <input 
+            type="date" name="birthday" 
+          >
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
          <input 
            type="date" 
            name="birthday" 
          >
```

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,152 @@
...
+        </label>
+
+        <label class="label">
+          I accept the term of the agreement
+          <input type="checkbox" name="acceptTerm"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="fieldset">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
        </label>
      </fieldset>
      
      <fieldset class="fieldset">
```

File: src/index.html [hunk 6]
```diff
@@ -1,17 +1,152 @@
...
+          Full date of birth: 
+          <input 
+            type="date" name="birthday" 
+          >
+        </label>
+
+        <label class="label">
+          I accept the term of the agreement
+          <input type="checkbox" name="acceptTerm"
+          >
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
          <input type="checkbox" name="acceptTerm">
```

File: src/index.html [hunk 7]
```diff
@@ -1,17 +1,152 @@
...
+          <input 
+            type="password"
+            name="password" 
+            minlength="5" 
+            maxlength="9"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="fieldset">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
        </label>
      </fieldset>
      
      <fieldset class="fieldset">
```

File: src/index.html [hunk 8]
```diff
@@ -1,17 +1,152 @@
...
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+
+          <label>
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]: no space between parent and child
```suggestion
        <label class="label">
          <label>
```

File: src/index.html [hunk 9]
```diff
@@ -1,17 +1,152 @@
...
+          <label> 
+            <input 
+              type="radio" 
+              name="cats" 
+              value="No" 
+            > No
+          </label>
+
+        </label>
+        <label class="label">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
          </label>
        </label>
        
        <label class="label">
```

File: src/index.html [hunk 10]
```diff
@@ -1,17 +1,152 @@
...
+          </select>
+        </label>
+
+        <label class="label">
+          How do you rate your work? 
+          <input type="range" name="rateWork" >
+        </label>
+
+      </fieldset>
+      <fieldset class="fieldset">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
        </label>
      </fieldset>
      
      <fieldset class="fieldset">
```

File: src/index.html [hunk 11]
```diff
@@ -1,17 +1,152 @@
...
+
+        <label class="label">
+          Would you recommend us?
+          <select name="reccomendation">
+            <option name="yes">yes</option>
+            <option name="no">no</option>
+          </select>
+        </label>
+        
+      </fieldset>
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html]:
```suggestion
        </label>
      </fieldset>
```

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]

**Review by Volodymyr-Melnyk-Fron10der — APPROVED** [review]
Nice job👍

**Review by DarkMistyRoom — APPROVED** [review]

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**